.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 notifications:
   slack:
     rooms:
-      - undeadbaby:$SLACK_KEY#random
+    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
```

Comment:
Yeah, I understood the risks. I'll dig some more now I have the correct
documentation.

On Wed, 27 Jul 2016, 16:14 laynecsmall, <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/operationzombie/Embedded-devices/pull/2#discussion_r72378722>
> :
>
> > @@ -8,6 +8,9 @@ install:
> >  notifications:
> >    slack:
> >      rooms:
> > -      - undeadbaby:$SLACK_KEY#random
> > +    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
>
> The travis environment variables are documented here:
>
>
> https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings
>
> The page on which the variables are currently set is here:
> https://travis-ci.org/operationzombie/apache-flask/settings (for the
> apache flask repo at least)
> the docker stuff is definitely working
>
> If you don't want to dig into this any more, I'll merge the PR, but if
> this had been something like an aws access key we'd be being billed for
> bitcoin mining right about now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/operationzombie/Embedded-devices/pull/2/files/e8360690be58cb5f7b920e5dcbda16e808fad5b8#r72378722>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARK0l7ch1f6sHEfRo1U_M9AQixMBIwGPks5qZts4gaJpZM4JVr6v>
> .
>